system-variables: add datadir and license (#5761)
pingcap/docs: system-variables.md

- Default value: ON
- Controls whether statements should automatically commit when not in an explicit transaction. See [Transaction Overview](/transaction-overview.md#autocommit) for more information.

### datadir

- Scope: NONE
- Default value: /tmp/tidb

> - This variable indicates the location where data is stored.

Reviewer: ```suggestion
- This variable indicates the location where data is stored. This can be a local path or point to a PD server in case the data is stored on TiKV.
```

> - Typically the value when be in the format of `ip_address:port`, indicating the PD server that TiDB connected to on startup.

Reviewer 2: ```suggestion
- Typically, the value in the format of `ip_address:port` indicates the PD server that TiDB connects to on startup.
```

Author: I made a small edit to your change. Combining with Daniel's feedback we can just say "A value in the format.." since above it clarifies it can be a local path.

- Default value：1000
- Controls the maximum recursion depth in Common Table Expressions.

### datadir

- Scope: NONE
- Default value: /tmp/tidb

> - This variable indicates the location where data is stored. This can be a local path or point to a PD server in case the data is stored on TiKV.

Reviewer 2: ```suggestion
- This variable indicates the location where data is stored. This location can be a local path or point to a PD server if the data is stored on TiKV.
```
